packages/storybook/stories/VideoEffects.stories.mdx

```diff
@@ -0,0 +1,53 @@
+import { Meta } from '@storybook/addon-docs';
+import { DetailedBetaBanner } from './BetaBanners/DetailedBetaBanner';
+import { Stack } from '@fluentui/react';
+import { overviewPageImagesStackStyle } from './constants';
+
+<Meta
+  id="videoeffects"
+  title="Concepts/Video Effects"
+  parameters={{ previewTabs: { canvas: { disable: true, hidden: true } } }}
+/>
+
+## Blurred and Custom Backgrounds
+
+<DetailedBetaBanner version={'1.5.1-beta.5'} />
+
+Azure Communication Services UI Library is adding support for Blurred and Custom Backgrounds.
+With this new feature, users can blur the user background or change the background to a provided default image gallery option.
+Contoso can set their own custom images that the end user can then set as their background, allowing for greater customization of the calling experience.
+Enabling this feature will enable greater privacy and allow for greater focus on the person by removing the existing background behind the user.
```

Comment:
```suggestion
Enable this feature to enhance privacy and to also direct focus on the individual by removing their existing  "real" background with a custom background or effect.
```

Feel free to rephrase "real". I was trying to define a difference between a custom and a background from the actual camera.